core/http-auth-aws-crt/pom.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License").
+  ~ You may not use this file except in compliance with the License.
+  ~ A copy of the License is located at
+  ~
+  ~  http://aws.amazon.com/apache2.0
+  ~
+  ~ or in the "license" file accompanying this file. This file is distributed
+  ~ on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+  ~ express or implied. See the License for the specific language governing
+  ~ permissions and limitations under the License.
+  -->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>software.amazon.awssdk</groupId>
+        <artifactId>core</artifactId>
+        <version>2.20.2-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>http-auth-aws-crt</artifactId>
+    <name>AWS Java SDK :: HTTP Auth AWS CRT</name>
+    <description>
+        The AWS SDK for Java - Auth module holds the AWS Common Runtime based implementations that are used for
+        authentication with HTTP services
+    </description>
+    <!-- TODO: this is not present in all pom.xml files. Is there a reason, or just a miss -->
+    <url>https://aws.amazon.com/sdkforjava</url>
```

Comment:
Probably just a miss.. Unless it's inherited from the parent?

Reply:
Hmm, doesn't seem to be inherited. https://github.com/aws/aws-sdk-java-v2/blob/master/http-client-spi/pom.xml doesn't have it and https://search.maven.org/artifact/software.amazon.awssdk/http-client-spi doesn't have _Home page_ link. Compared to https://search.maven.org/artifact/software.amazon.awssdk/auth which does.

Reply:
removed these TODOs and another one for HTTP v/s Http as we use HTTP in other modules already.

Reply:
Must just be a miss, then!